Gemfile

```diff
@@ -16,6 +16,8 @@ gem "draper"
 gem "enumerize"
 gem "filterrific", github: "metikular/filterrific", branch: "fix/nested-array"
 gem "friendly_id"
+gem "good_job"
```

Comment:
We could include the good_job dashboard if @wintermeyer wants to have some inspection in the scheduler.

Reply:
I'd suggest to do this in a seperate PR